.github/workflows/release.yml

```diff
@@ -8,14 +8,13 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - id: tag
         uses: freckle/haskell-tag-action@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - if: steps.tag.outputs.tag
-        uses: freckle/stack-upload-action@v2
```

Comment:
`stack-upload-action` looks for specifically `stack-lts-*.yaml` and so misses our `stack-ltsX.yaml` files and released v1.1.3.1 with `stack.yaml` and pretty high lower bounds. Now that `stack` supports `HACKAGE_API_KEY` we don't need a special action and just use `stack upload` directly, with an explicit `STACK_YAML` to set the lower bounds.

Once v1.1.3.2 is out I will either do a revision to v1.1.3.1 or just deprecate it.